app/javascript/components/resumebuilder/multiStepForm/formPages/About.jsx

```diff
@@ -18,11 +18,23 @@ import camelcaseKeysDeep from "camelcase-keys-deep";
 import decamelizeKeysDeep from "decamelize-keys-deep";
 
 const aboutSchema = {
```

Comment:
change this to aboutSchemaWQns and newAbout back to aboutSchema for readability.